compiler/src/main/kotlin/io/github/apl_cornell/viaduct/backends/aby/ABY.kt

```diff
@@ -27,10 +28,10 @@ sealed class ABY(val server: Host, val client: Host) : Protocol() {
     override val arguments: Map<String, Value>
         get() = mapOf("server" to HostValue(server), "client" to HostValue(client))
 
-    override fun authority(hostTrustConfiguration: HostTrustConfiguration): Label {
-        val combined = hostTrustConfiguration(server).interpret() join hostTrustConfiguration(client).interpret()
+    override fun authority(): Label {
+        val combined = LabelLiteral(server).interpret() join LabelLiteral(client).interpret()
```

Comment:
Don't use `.interpret()`. You have defined a function `Host.label`. Use that instead.